run-server.sh

```diff
@@ -1,10 +1,10 @@
 #!/bin/sh -e
 
-if [ "z$HOST" = "z" ]; then
+if [ -z "$HOST" ]; then
```

Comment:
I think this only works in bash, as that would support `[[ ]]`, which is an alias for `test`.